node/src/asset_cleanup.rs

```diff
@@ -0,0 +1,416 @@
+use crate::assets::clean_db;
+use crate::config::ParticipantsConfig;
+use crate::db::{DBCol, SecretDB, EPOCH_ID_KEY};
+use crate::primitives;
+use crate::providers::ecdsa::presign::PresignOutputWithParticipants;
+use crate::providers::ecdsa::triple::{PairedTriple, TRIPLE_STORE_DOMAIN_ID};
+use mpc_contract::primitives::domain::DomainId;
+use mpc_contract::primitives::key_state::EpochId;
+use serde::{self, Deserialize, Serialize};
+use std::sync::Arc;
+
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
+pub struct EpochData {
+    pub epoch_id: EpochId,
+    pub participants: ParticipantsConfig,
+}
+
+/* database helpers */
+enum EpochDataWrapper {
+    Legacy(EpochId),
+    Current(EpochData),
+}
+
+fn get_epoch_data(db: &Arc<SecretDB>) -> anyhow::Result<Option<EpochDataWrapper>> {
+    let Some(db_res): Option<Vec<u8>> = db.get(DBCol::EpochData, EPOCH_ID_KEY)? else {
+        return Ok(None);
+    };
+    if let Ok(epoch_data) = bincode::deserialize::<EpochData>(&db_res) {
+        return Ok(Some(EpochDataWrapper::Current(epoch_data)));
+    }
```

Comment:
Unrelated: I'm surprised we don't have a wrapper on the database level which also does the serialization. Also I notice we sometimes use JSON and sometimes bincode for serializing our data. Why do we use different serialization formats for different columns?

Reply:
Yeah, that's an excellent observation. Would love to fix this in a refactor.